scripts/make_acousticbrainz_genre.py

```diff
@@ -0,0 +1,78 @@
+import argparse
+import csv
+import hashlib
+import json
+import os
+import itertools
+import re
+
+
+acousticbrainz_genre_INDEX_PATH = '../mirdata/indexes/acousticbrainz_genre_index.json'
+ACOUSTICBRAINZ_GENRE_ANNOTATION_SCHEMA = ['JAMS']
+
+
+def md5(file_path):
+    """Get md5 hash of a file.
+    Parameters
+    ----------
+    file_path: str
+        File path.
+    Returns
+    -------
+    md5_hash: str
+        md5 hash of data in file_path
+    """
+    hash_md5 = hashlib.md5()
+    with open(file_path, 'rb') as fhandle:
+        for chunk in iter(lambda: fhandle.read(4096), b''):
+            hash_md5.update(chunk)
+    return hash_md5.hexdigest()
+
+
+def make_acousticbrainz_genre_index(data_path):
+    index = 0
+    datasets = ['tagtraum', 'allmusic', 'lastfm', 'discogs']
+    dataset_types = ['validation', 'train']
+    f = open(acousticbrainz_genre_INDEX_PATH, 'w')
+    f.write('{\n')
+    for dataset, dataset_type in itertools.product(datasets, dataset_types):
+        tsv_file = open(os.path.join(data_path, "acousticbrainz-mediaeval-" + dataset + "-" + dataset_type + ".tsv"))
+        read_tsv = csv.reader(tsv_file, delimiter="\t")
+        next(read_tsv, None)
+        read_tsv_list = list(read_tsv)
+        for line, row in enumerate(read_tsv_list):
+            mbid = ""
+            track_id = dataset + '#' + dataset_type
+            for i, r in enumerate(row):
+                track_id = track_id + '#' + r
+                if i == 0:
+                    mbid = r
+            ann_path = os.path.join(data_path, "acousticbrainz-mediaeval-" + dataset_type, mbid[:2], mbid + ".json")
+            f.write('  \"%s\": {\n' % (track_id,))
```

Comment:
Is there a reason to write json by hand here? I think it would be safer to write it as json, unless there's a specific reason.

Reply:
Yes. The full index is massive, and python throws an exception due to that. The rest of datasets have been developed with `JSON` package